content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -151,10 +164,11 @@ Install containerd:
 {{< tabs name="tab-cri-containerd-installation" >}}
 {{% tab name="Linux" %}}
 
-1. Install the `containerd.io` package from the official Docker repositories. 
-Instructions for setting up the Docker repository for your respective Linux distribution and 
-installing the `containerd.io` package can be found at 
-[Install Docker Engine](https://docs.docker.com/engine/install/#server).
+1. Install the `containerd.io` package from the [official containerd website](
+   https://containerd.io/downloads/).Instructions for setting up the Docker
```

Comment:
You don't need to set up the **Docker** package repository, if installing from the containerd.io.

So either you install the "containerd.io" package from Docker, or you install the "containerd"
package from your distribution (`apt install containerd`) or download it directly from containerd.io

If you want to keep both options (docker.com and containerd.io), they need to be clearly separated.

Reply:
+1

Reply:
@afbjorklund thank you.

Reply:
we should remove the docker repository from the containerd steps and leave the user two options:
- following the guide in https://containerd.io/downloads/ and https://containerd.io/docs/getting-started/#starting-containerd
- use the package manager of the Linux distro

Reply:
Would it be possible to supply deb packages for containerd, the same way as for "cri-tools" and "kubernetes-cni" ?

From apt.kubernetes.io, that is.

Reply:
The package management in k8s needs to be community owned. Currently Google still holds the keys. But also some reorganization of the package repositories might be required for better version flexibility.

The containerd maintainers can open a discussion about that with the k8s SIG Release. In the meantime we should document the best option, IMO.

Reply:
Maybe CRI and CNI should be decoupled from the packaging as well, they would be less outdated that way I think.

Reply:
@PranshuSrivastava I recommend following the approach outlined in https://github.com/kubernetes/website/pull/30882#discussion_r816261724

@afbjorklund, I'm keen to keep the focus of this PR on just the set of changes we'd make to this page so that the v1.24 docs make sense after the dockershim component is removed. Once this merges, we can consider follow-up PRs that make further improvements.

As the proverb doesn't quite say, the perfect is the enemy of the merged.

Reply:
I think changing from docker.com to containerd.io would be enough, go with the tarball if that is all there is.

Reply:
Can we keep containerd installation instructions as they are as they're not actually related to the purpose of this PR. We can update them in a separate PR I believe.

Reply:
After we remove the dockershim, the current instructions are too likely to confuse people. I think fixing that does align with the intent of this PR.